.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,38 @@
+name: Jupyter notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test Jupyter notebooks on ${{ matrix.python-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
```

Comment:
you can add `fail-fast: false`, by default actions kills all cases of the matrix if one fails.

Reply:
Would the syntax for this simply be akin to:
```
name: Jupyter notebooks

fail-fast: false

on:
 ...
```

Reply:
yes, but inside strategy, see
https://github.com/devitocodes/devito/blob/b32cb4fcced388bbbaa31b293ab0b19384b87dee/.github/workflows/pytest-core-nompi.yml#L29

Reply:
Resolved in [`98fd6e1`](https://github.com/devitocodes/devito/pull/1069/commits/98fd6e144a9270905e701619e442d14613594b7f)